src/py/_pyodide/_core_docs.py

```diff
@@ -1027,10 +1027,14 @@ def children(self) -> Sequence["JsDomElement"]:
     def appendChild(self, child: "JsDomElement") -> None:
         pass
 
-    def addEventListener(self, event: str, listener: Callable[[Any], None]) -> None:
+    def addEventListener(
+        self, event: str, listener: Callable[[Any], None], kwargs: Any
```

Comment:
```suggestion
        self, event: str, listener: Callable[[Any], None], **kwargs: Any
```

Reply:
I think this does actually want to be `kwargs` (or something like `options`) if we want to pass along a single object with all the kwargs passed as attributes, yes?

Reply:
Actually that might not be what we want... apparently there are cases where you'd want to pass multiple separate arguments which might be objects or booleans (i.e. `options` and `wantsUntrusted`), which makes me thing we should use `*args`?

Reply:
It's the same to say `{capture: true}` as to say `true` there, so I don't see the reason to use the unnamed version, it's strictly less readable. If you want to be 100% accurate I think you'd need something like the following:
```
@override
def addEventListener(
    self, event, handler: Callable[[Event], None], capture: bool = False, /
):
    ...


@override
def addEventListener(
    self,
    event: str,
    handler: Callable[[Event], None],
    /,
    *,
    capture: bool = False,
    once: bool = False,
    passive: bool = False,
    signal: JsProxy | None
):
    ...

def addEventListener(self, event, handler, /, *args, **kwargs):
    raise NotImplementedError
```